src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1694,6 +1718,97 @@ def _get_or_add_extension(extension_name, extension_module, update=False):
     return True
 
 
+def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id, resource_group_name):
+    # log analytics workspaces cannot be created in WCUS region due to capacity limits so mapped to EUS per discussion with log analytics team
+    AzureLocationToOmsRegionCodeMap = {
+        "eastus": "EUS",
+        "westeurope": "WEU",
+        "southeastasia": "SEA",
+        "australiasoutheast": "ASE",
+        "usgovvirginia": "USGV",
+        "westcentralus": "EUS",
+        "japaneast": "EJP",
+        "uksouth": "SUK",
+        "canadacentral": "CCA",
+        "centralindia": "CIN",
+        "eastus2euap": "EAP"
+    }
+    AzureRegionToOmsRegionMap = {
+        "australiaeast": "australiasoutheast",
+        "australiasoutheast": "australiasoutheast",
+        "brazilsouth": "eastus",
+        "canadacentral": "canadacentral",
+        "canadaeast": "canadacentral",
+        "centralus": "eastus",
+        "eastasia": "southeastasia",
+        "eastus": "eastus",
+        "eastus2": "eastus",
+        "japaneast": "japaneast",
+        "japanwest": "japaneast",
+        "northcentralus": "eastus",
+        "northeurope": "westeurope",
+        "southcentralus": "eastus",
+        "southeastasia": "southeastasia",
+        "uksouth": "uksouth",
+        "ukwest": "uksouth",
+        "westcentralus": "eastus",
+        "westeurope": "westeurope",
+        "westus": "eastus",
+        "westus2": "eastus",
+        "centralindia": "centralindia",
+        "southindia": "centralindia",
+        "westindia": "centralindia",
+        "koreacentral": "southeastasia",
+        "koreasouth": "southeastasia",
+        "francecentral": "westeurope",
+        "francesouth": "westeurope"
+    }
```

Comment:
Shouldn't these data be either:
1. In the SDK;
2. Returns by the service endpoint?

Reply:
This mapping specific default workspace logic of the log analytics. These data not available either in SDK or Service endpoint.

Reply:
ok. it's fine to add them here for now. but in the long-term, the configuration data and service metadata should be served in the service endpoint or in the SDK.